LICENSE.md

```diff
@@ -0,0 +1,15 @@
+# The ISC License (ISC)
+
+**Copyright &copy; 2021, JT \<jt@gosquared.com>**
```

Comment:
Oh, I forgot to comment this - the copyright assignment here should be "Go Squared Ltd." rather than me personally - but I'm happy to make sure that's set straight when we merge this